### PR TITLE
ROC-3821 output simplification

### PR DESCRIPTION
**Pull request by andrewwa-kainos**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -18,5 +18,5 @@ resource "azurerm_template_deployment" "action-group" {
 }
 
 output "action_group_name" {
-  value = "${azurerm_template_deployment.action-group.parameters.actionGroupName}"
+  value = "${var.action_group_name}"
```
Comment (timja-kainos) [inline main.tf]: any reason to do this?

Comment (andrewwa-kainos) [reply main.tf]:
```Error: Error running plan: 1 error(s) occurred:

* module.cmc-bulk-print-fail-action-group.output.action_group_name: Resource 'azurerm_template_deployment.action-group' does not have attribute 'parameters.actionGroupName' for variable 'azurerm_template_deployment.action-group.parameters.actionGroupName'```

Comment (andrewwa-kainos) [reply main.tf]: Also just more straightforward and easier to understand. The only downside would be if at some point we mutated it from `var` to `parameters` and the `var` value is no longer the correct one but I think that's negligible.

Comment (timja-kainos) [reply main.tf]: ok

**Pull request merged**